src/main/java/mate/academy/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
```suggestion
@Getter
@Setter
```

Don't use Data annotation, read lecture materials about the issues

Reply:
> Don't use Data annotation, read lecture materials about the issues

Where is this mentioned, because I couldn’t find it in the video or in the theory?

Reply:
<img width="1421" alt="Screenshot 2024-06-20 at 09 55 58" src="https://github.com/Leanchers/online-book-store/assets/14053156/8152d3e5-8494-4784-aebf-088a852b1476">

Reply:
I think you haven't learned this materials yet, but it's good to know. So you can read this [article](https://medium.com/@miguelangelperezdiaz444/the-hidden-dangers-of-lombok-annotations-in-your-java-code-what-you-need-to-know-8acdce2d6b89)